# KondoServoCommandMsg.msg

component_lib/GenericComponentMsg component_property

uint8 current_limit
uint16 command_degree
uint8 speed
uint8 stretch
uint8 temp_limit
bool free
# see the definitions in component_lib.h
